cypress/e2e/capidle/isGuessCorrect.cy.js

```diff
@@ -0,0 +1,28 @@
+/// <reference types="cypress" />
```

Comment:
Are you able to rename this file to `is-guess-correct` to keep it consistent with the other test files?

Reply:
Same with the JSON files in `puzzles`

Reply:
good spotting, let's be consistent with naming